client/project-router.js

```diff
@@ -92,10 +96,12 @@ const ProjectRouter = () => {
     };
   });
 
+  const downloadBasename = basename.replace(/\/edit/, '');
```

Comment:
This is a _very_ liberal regex. Does that have the potential to screw future us? Should we at least make sure it appears at the end?

Reply:
```suggestion
  const downloadBasename = basename.replace(/\/edit($|\/)/, '');
```